packages/contracts-bedrock/test/dispute/FaultDisputeGame.t.sol

```diff
@@ -492,6 +492,100 @@ contract FaultDisputeGame_Test is FaultDisputeGame_Init {
         gameProxy.step(8, true, absolutePrestateData, hex"");
     }
 
+    /// @dev Tests that successfully step with true attacking claim when there is a true defend claim(claim5) in the middle of the dispute game.
+    function test_stepAttackDummyClaim_defendTrueClaimInTheMiddle() public {
```

Comment:
```suggestion
    function test_stepAttackDummyClaim_defendTrueClaimInTheMiddle_succeeds() public {
```

Reply:
Fixed in https://github.com/ethereum-optimism/optimism/pull/9986/commits/a6d5f0817b1416356ae707bfc2b422d35bb1571d